### PR TITLE
add CFF commentary

### LINKED ISSUE #29: Feature: CycleFreeFluxes
https://academic.oup.com/bioinformatics/article/31/13/2159/195895

### PR DESCRIPTION
**Pull request by exaexa**
closes #29  (in a very convenient way)

### CODE REVIEW

**Review by stelmo — COMMENTED** [review]
Need to fix external fluxes + one or two more things from paper. I can do this next week

**Comment by exaexa** [comment]
OK we might want to add some explicit way to fix certain fluxes (the exchanges). That way it substantiates a separate function.

File: src/frontend/parsimonious.jl [hunk 1]
```diff
@@ -99,6 +99,8 @@ parsimonious solutions (such as the general lack of unnecessary loops). On the
 other hand, like with plain flux balance analysis there is no strong guarantee
 of uniqueness of the solution.
 
+Notably, this is equivalent to the CycleFreeFlux algorithm.
+
```
Comment (exaexa) [inline src/frontend/parsimonious.jl:103]: asdasdad
```suggestion
qawefdsadfsdfs
```

**Comment by exaexa** [comment]
obsoleted

**Pull request closed without merging**